inst/markdown/compareScenarios2/cs2_06_energy_services.Rmd

```diff
@@ -5,57 +5,60 @@
 ```{r FE demand from buildings by carriers}
 tot <- "FE|Buildings"
 items <- c(
-  "FE|Buildings|non-Heating|Electricity|Conventional",
-  "FE|Buildings|Heating|Solids",
-  "FE|Buildings|Heating|Liquids",
-  "FE|Buildings|Heating|Gases",
-  "FE|Buildings|Heating|District Heating",
+  "FE|Buildings|non-Heating|Electricity|Conventional",  
+  "FE|Buildings|Heating|Electricity|Heat pump",  
+  "FE|Buildings|Heating|District Heating",  
   "FE|Buildings|Heating|Electricity|Resistance",
-  "FE|Buildings|Heating|Electricity|Heat pump",
-  "FE|Buildings|Heating|Hydrogen")
+  "FE|Buildings|Heating|Hydrogen",  
+  "FE|Buildings|Heating|Gases",
+  "FE|Buildings|Heating|Liquids",
+  "FE|Buildings|Heating|Solids",
+  NULL)
 showAreaAndBarPlots(data, items, tot, orderVars = "user", scales = "fixed")
 ```
 
 ### UE mix
 ```{r UE demand from buildings by carriers}
 tot <- "UE|Buildings"
-items <- c(
-  "UE|Buildings|non-Heating|Electricity|Conventional",
-  "UE|Buildings|Heating|Solids",
-  "UE|Buildings|Heating|Liquids",
-  "UE|Buildings|Heating|Gases",
-  "UE|Buildings|Heating|District Heating",
+  "UE|Buildings|non-Heating|Electricity|Conventional",  
```

Comment:
You deleted too much here. `items <- c(` ...

Reply:
oops :) thanks for the catch

Reply:
Check out pre-commit.ci (see below in list of checks). It reveals the error, because it results in non-parsable R code
https://results.pre-commit.ci/run/github/333151188/1702482231.YLGojE8XSymCWE7yhH-xhw

Reply:
thanks! Also spotted another bug.. patches have been pushed